lib/generate/consents.rb

```diff
@@ -98,5 +119,15 @@ def create_consent_given_needs_triage(count)
         )
       end
     end
+
+    def validate_programme_and_session(programme, session)
+      if session
+        if session.programmes.exclude?(programme)
+          raise "Session does not support programme #{programme.type}"
+        end
+      elsif programme.sessions.none? { |s| s.location.school? }
```

Comment:
```suggestion
      elsif programme.sessions.none? { it.location.school? }
```

I'm not sure `s` is any more meaningful than `s`.